pallets/nfts/src/benchmarking.rs

```diff
@@ -571,40 +592,65 @@ benchmarks_instance_pallet! {
 	}
 
 	approve_transfer {
+		let i in 0 .. 1;
+
 		let (collection, caller, _) = create_collection::<T, I>();
 		let (item, ..) = mint_item::<T, I>(0);
 		let delegate: T::AccountId = account("delegate", 0, SEED);
 		let delegate_lookup = T::Lookup::unlookup(delegate.clone());
 		let deadline = BlockNumberFor::<T>::max_value();
-	}: _(SystemOrigin::Signed(caller.clone()), collection, item, delegate_lookup, Some(deadline))
+		let maybe_item = if i == 0 {
+			None
+		} else {
+			Some(item)
```

Comment:
Same for others below, small optimisation

Reply:
This is not possible. I did have this before but because we check if the collection items of an account is empty or not to approve_transfer, we will need to mint for both cases.